config/locales/exceptions.en.yml

```diff
@@ -462,12 +498,24 @@ en:
         code: 'VET360_CORE101'
         detail: 'You do not have access to perform the requested operation'
         status: 400
+      VET360_CORE102:
+        <<: *external_defaults
+        title: Data integrity violation
+        code: 'VET360_CORE102'
+        detail: 'The operation could not be fulfilled due to a data integrity violation.'
+        status: 400
       VET360_CORE103:
         <<: *external_defaults
         title: Not Found
         code: 'VET360_CORE103'
         detail: 'The Vet360 id could not be found'
         status: 404
+      VET360_CORE104:
+        <<: *external_defaults
+        title: No changes detected
+        code: 'VET360_CORE104'
+        detail: 'Your request was received and processed without error, however no differences were detected between current data and the data you sent. This message is informational only, please verify your request if you believe you sent actual changes that should be applied.'
```

Comment:
I find this one randomly long `detail:` amusing. Super terse _everywhere_ else and then suddenly this extremely verbose message. "This message is information only"? What else would it be? A chatbot? :-)